src/net/sourceforge/kolmafia/utilities/HashMultimap.java

```diff
@@ -11,14 +10,14 @@
  */
 public class HashMultimap<V> extends HashMap<Integer, ArrayList<V>> {
   public final void put(int key, V value) {
-    Integer okey = IntegerPool.get(key);
+    Integer okey = key;
```

Comment:
I initially thought "okay", but it is "o-key", isn't it.

Reply:
A lot of these can be removed because they'll be inferred using autoboxing.

The trick is knowing which ones are safe to remove, because there is Mafia code that treats ints and Integers differently.